contrib/DEVELOPMENT_WORKFLOW.md

```diff
@@ -1,159 +1,111 @@
 # Bittensor Development Workflow
 
-## Table of contents
+## Table of Contents
 
-- [Bittensor Development Workflow](#bittensor-development-workflow)
-  - [Main Branches](#main-branches)
-  - [Development Model](#development-model)
-      - [Feature Branches](#feature-branches)
-      - [Release Branches](#release-branches)
-      - [Hotfix Branches](#hotfix-branches)
-  - [Git Operations](#git-operations)
-      - [Creating a Feature Branch](#creating-a-feature-branch)
-      - [Merging Feature Branch into Staging](#merging-feature-branch-into-staging)
-      - [Creating a Release Branch](#creating-a-release-branch)
-      - [Finishing a Release Branch](#finishing-a-release-branch)
-      - [Creating a Hotfix Branch](#creating-a-hotfix-branch)
-      - [Finishing a Hotfix Branch](#finishing-a-hotfix-branch)
-  - [Continuous Integration (CI) and Continuous Deployment (CD)](#continuous-integration-ci-and-continuous-deployment-cd)
-  - [Versioning and Release Notes](#versioning-and-release-notes)
-  - [Pending Tasks](#pending-tasks)
+- [1. Main Components](#1-main-components)
+    - [Bittensor Protocol](#bittensor-protocol)
+    - [Neurons](#neurons)
+    - [Subtensor](#subtensor)
+    - [Metagraph](#metagraph)
+- [2. Developer Notes](#2-developer-notes)
+    - [Notes for Pull Request](#notes-for-pull-request)
+    - [Notes for Releasing](#notes-for-releasing)
+    - [Notes for Logging](#notes-for-logging)
+        - [Logging Setup](#logging-setup)
+        - [Logging Messages](#logging-messages)
+        - [Logging Variables](#logging-variables)
+        - [Logging Exceptions](#logging-exceptions)
 
-## Main Branches
 
-Bittensor's codebase consists of two main branches: **master** and **staging**.
 
-**master**
-- This is Bittensor's live production branch, which should only be updated by the core development team. This branch is protected, so refrain from pushing or merging into it unless authorized.
+# 1. Main Components
```

Comment:
NACK: Bittensor protocol should not be described here as it's subject to change.  This document should stand alone regardless of how the underlying protocol changes. This is specifically for developer workflow, not documentation.